utils/src/ooinstall/cli_installer.py

```diff
@@ -10,7 +10,7 @@
 from pkg_resources import parse_version
 from ooinstall import openshift_ansible
 from ooinstall.oo_config import OOConfig
-from ooinstall.oo_config import OOConfigInvalidHostError
+from ooinstall.models import InvalidHostError
 from ooinstall.oo_config import Host, Role
```

Comment:
Still not sure how this isn't erroring. Shouldn't `Host` and `Role` be imported from `ooinstall.models` now?